lib/atc.js

```diff
@@ -16,7 +16,7 @@ atc.connected = false
 atc.holdMessages = []
 atc.id = crypto.randomBytes(16).toString('hex')
 atc.sentMessages = []
-atc.type = 'server'
```

Comment:
Why default to server, should this not be a parameter from whoever is calling this library?